Solutions/PaloAlto-PAN-OS/Playbooks/PaloAltoPlaybooks/PaloAlto-PAN-OS-BlockURL-EntityTrigger/readme.md

```diff
@@ -0,0 +1,217 @@
+# PaloAlto-PAN-OS-BlockURL
+
+ ## Summary
+
+This playbook allows blocking/unblocking URLs in PaloAlto, using **predefined address group**. This allows to make changes on predefined address group, which is attached to security policy rule. 
+When a new Sentinel incident is created, this playbook gets triggered and performs below actions:
+
+1. An adaptive card is sent to the SOC channel providing Incident information, URL address, list of existing security policy rules in which URL is a member of and provides an option to Block/Unblock URL Address by adding/removing it to/from the predefined address group.
+
+2. The SOC can act on risky URL based on the information provided in the adaptive card, or ignore.
+
+
+![PaloAlto-PAN-OS-BlockURL](./images/designerscreenshot_lighttheme.PNG)<br>
+
+**This is the adaptive card SOC will receive when playbook is triggered for each risky URL for taking actions like block/unblock/ignore ::**<br><br>
+![Adaptive Card example](./images/AdaptiveCardtoBlockorUnblock.PNG)<br>
+
+**This is the consolidate adaptive card about the summary of actions taken on URL and the incident configuration ::**<br><br>
+![Consolidated Adaptive Card example](./images/SummarizedAdaptiveCard.PNG)<br>
+
+### Prerequisites 
+1. PaloAlto connector needs to be deployed prior to the deployment of this playbook under the same subscription. Relevant instructions can be found in the connector doc page.
+2. Generate an API key.[Refer this link on how to generate the API Key](https://docs.paloaltonetworks.com/pan-os/10-1/pan-os-panorama-api/get-started-with-the-pan-os-xml-api/get-your-api-key)
+3. Address group should be created for PAN-OS and this should be used while creating playbooks.
+
+
+
+### Deployment instructions 
+1. Deploy the playbook by clicking on "Deploy to Azure" button. This will take you to deploying an ARM Template wizard.
+
+
+[![Deploy to Azure](https://aka.ms/deploytoazurebutton)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2FAzure-Sentinel%2Fmaster%2FSolutions%2FPaloAlto-PAN-OS%2FPlaybooks%2FPaloAltoPlaybooks%2F
```

Comment:
Deploy button links doesn't seem to be working.

Reply:
It will work once the Pr is merged I have directly pointed them to master branch